plugins/lua/suspendmanager.lua

```diff
@@ -0,0 +1,131 @@
+local _ENV = mkmodule('plugins.suspendmanager')
+
+local utils = require('utils')
+local gui = require('gui')
+local overlay = require('plugins.overlay')
+local widgets = require('gui.widgets')
+
+--- Loop over all the construction jobs
+---@param fn function A function taking a job as argument
+function foreach_construction_job(fn)
+    for _, job in utils.listpairs(df.global.world.jobs.list) do
+        if job.job_type == df.job_type.ConstructBuilding then
+            fn(job)
+        end
+    end
+end
+
+function isKeptSuspended(job)
+    return suspendmanager_isKeptSuspended(job)
+end
+
+-- TODO: check that this is appropriate
+function isBuildingPlanJob(job)
```

Comment:
this code is duplicated in the cpp -- how about exposing that one and removing the Lua version?